spk/radarr/Makefile

```diff
@@ -1,22 +1,24 @@
 SPK_NAME = radarr
 SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 14
+SPK_REV = 15
 SPK_ICON = src/radarr.png
 
 REQUIRED_DSM = 5.0
-# Mono not supported on ppc platforms. C.f. cross/mono/Makefile and references therein for details.
-UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 
-DEPENDS = cross/busybox cross/curl cross/mediainfo cross/sqlite cross/radarr
+# .NET is not supported on PPC, ARM5 or x86
+UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARM5_ARCHES) $(x86_ARCHES)
```

Comment:
@publicarray Why not use compilation with Mono implementation for these architectures? (the same way as for Jackett if I remember well)

Reply:
I mean we can add it for one more version:

@ta264 

> I agree it's a bit of a hack but I think it's probably better than nothing. The mono builds in Radarr will be switched off in the not too distant future (in case you haven't realised, I'm a Radarr dev) - we're planning to release 3.1 but then I think the next release will be 4.0 which won't support mono. We'd love to not leave the non-docker-compatible synology users behind, but we're not willing to maintain mono builds to make that happen I'm afraid.

Reply:
Actually there shouldn't be any need since the package auto updates until it won't be supported anymore.